docker-compose.yml

```diff
@@ -32,6 +32,15 @@ services:
       - shared-file-system:/serialization-sanitizer:delegated
       - vt-cache:/build:delegated
       - vt-build:/build/vt:delegated
+    command:
+      /bin/bash -c "
+        pushd /vt &&
+        sed -i '/# Primary VT build/i add_definitions( \
```

Comment:
just inserting the necessary options with `sed` seems hackish, but it has big advantage of not requiring changes in `vt`